src/ofdm_internal.h

```diff
@@ -82,19 +82,55 @@ struct OFDM_CONFIG {
     float rs;  /* Modulation Symbol Rate */
     float ts;  /* symbol duration */
     float tcp; /* Cyclic Prefix duration */
-    float ofdm_timing_mx_thresh;
+    float timing_mx_thresh;
 
     int nc;  /* Number of carriers */
     int ns;  /* Number of Symbol frames */
     int bps;   /* Bits per Symbol */
     int txtbits; /* number of auxiliary data bits */
     int ftwindowwidth;
+
+    int samples_per_frame;
 };
 
 struct OFDM {
+    struct OFDM_CONFIG config;
+    
```

Comment:
I was wondering if we still need ```config``` embedded in the ofdm struct but I think we do - so we can return a ptr to it if someone wants to take a peek at the current config.  An alternative would be a getter that copies the config into a user supplier config struct.

Reply:
Hmm... A getter that copied the values into a user supplied struct might save memory in the Embedded version.

Reply:
Yes and maybe more intuitive than https://github.com/drowe67/codec2/blob/dr-ofdm-rm-statics/src/freedv_2020.c#L45

Reply:
But it's a fine point....what we have is pretty good :smile: I'm impressed with how easy this modem is to modify (e.g. this PR and the QAM work).  Shows we have a good design :+1: